binding.cc

```diff
@@ -1179,6 +1180,26 @@ namespace zmq {
     NanReturnValue(NanNew<String>(version_info));
   }
 
+#if ZMQ_VERSION_MAJOR >= 4
+   static NAN_METHOD(ZmqCurveKeypair) {
+    NanScope();
+
+    char public_key [41];
+    char secret_key [41];
+
+    int rc = zmq_curve_keypair( public_key, secret_key);
+    if (rc < 0) {
+      throw std::runtime_error(ErrorMessage());
```

Comment:
Actually I got this from other parts of the code. But looking at it again seems to me that ```        NanThrowError(ExceptionFromError());
``` could be more appropriate. Would this be ok?